docs/template.js

```diff
@@ -22,7 +22,6 @@ const head = `<meta charset="utf-8"/>
 <link rel="stylesheet" href="/css/normalize.css"/>
 <link rel="stylesheet" href="/css/base.css"/>
 <link rel="stylesheet" href="/css/Draft.css"/>
-<link rel="stylesheet" href="/app.css"/>
```

Comment:
why is this removed???

Reply:
@thomas88 all looks good except this??

Reply:
Great that you look over this diligently, I've made a mistake here.

The line was removed, as I was seeing following warning message in console and found that the file wasn't existing.

<img width="1438" alt="screen shot 2017-09-29 at 11 45 01" src="https://user-images.githubusercontent.com/1851359/31010774-4940f784-a50c-11e7-8163-860db2a47d5a.png">

However I didn't understand that template.js is also used in production, where app.css is the built version of the css.

It would be great to find a way to remove the warning in development. For now I'm adding the file to the template again.

Reply:
@thomas88 awesome thanks. I'm working on new [documentation btw, using gitbook](https://github.com/draft-js-plugins/draft-js-plugins/pull/899) so all of this will soon go away, and it'll be easier to write and read docs!